### PR TITLE
docs: examples page

### PR DESCRIPTION
**Pull request by salihozdemir**
**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

Create new page that shows examples and demos.

### Test plan (required)

Tests are not effected with this changes.

### Self Check before Merge

Please check all items below before review.

-   [x] Corresponding issues are created/updated or not needed
-   [x] Docs are updated/provided or not needed
-   [x] Examples are updated/provided or not needed
-   [x] TypeScript definitions are updated/provided or not needed
-   [x] Tests are updated/provided or not needed
-   [x] Changesets are provided or not needed

### CODE REVIEW

File: documentation/src/components/examples/show-case-card/index.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+                            onClick={() => setSelectedButton(b)}
+                            className={clsx(
+                                "example-lineer-gradient-button font-montserrat flex cursor-pointer items-center gap-2 rounded-full border border-solid border-[#F0F2F5] bg-[#F6F6F9] px-3 py-2 font-bold uppercase text-[#9696B4] hover:text-white",
+                                {
+                                    "example-active-lineer-gradient-button":
+                                        selectedButton === b,
+                                },
+                            )}
+                        >
+                            {b?.icon && <Icon className=" object-contain" />}
```
Comment (biskuvit) [inline documentation/src/components/examples/show-case-card/index.tsx:45]:
```suggestion
                            {Icon && <Icon className=" object-contain" />}
```

**Review by biskuvit — APPROVED** [review]

**Review by omeraplak — APPROVED** [review]

File: documentation/src/assets/examples.ts [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+        button: {
+            text: "Read Blog",
+            link: "https://refine.dev/blog/customizable-issue-tracker-with-refine-and-supabase",
+        },
+    },
+    {
+        title: "Next.js E-commerce App",
+        description:
+            "An e-commerce client developed with Strapi & Chakra UI & <strong>refine</strong>",
+        image: "/examples/issue-tracker.png",
```
Comment (omeraplak) [inline documentation/src/assets/examples.ts:110]:
```suggestion
        image: "/examples/issue-tracker.png",
```
wrong screenshot

File: documentation/src/components/examples/example-card/index.tsx [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+export const ExampleCard: React.FC<ExampleCardProps> = ({ example }) => {
+    const { button, description, image, title, isExternal } = example;
+
+    const { reward: leftReward, isAnimating: leftIsAnimating } = useReward(
+        "leftReward",
+        "emoji",
+        {
+            emoji: ["🔥", "⭐", "❤️"],
+            // emoji: ["❤️", "💙", "💜", "🧡", "💖"],
```
Comment (ozkalai) [inline documentation/src/components/examples/example-card/index.tsx:21]: we can remove unused imojis

File: documentation/src/css/custom.css [hunk 4]
```diff
@@ -359,19 +359,19 @@
...
 }
 
 .refine-docsearch-button .DocSearch-Button-Placeholder {
-    @apply font-montserrat font-semibold text-xs text-[#2A2A42];
+    @apply font-montserrat text-xs font-semibold text-[#2A2A42];
 }
 
 .refine-docsearch-button .DocSearch-Search-Icon {
-    @apply w-3 h-3 stroke-[2px] -mt-px;
+    @apply -mt-px h-3 w-3 stroke-[2px];
```
Comment (ozkalai) [inline documentation/src/css/custom.css:374]: Is there a class named `-mt-px`?  I thought it has typo, if not please resolve my comment :)

File: documentation/src/assets/examples.ts [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+        image: "/examples/email-subscription.png",
+        button: {
+            text: "READ BLOG",
+            link: "https://refine.dev/blog/e-mail-subscription-panel-with-refine",
+        },
+    },
+    {
+        title: "INTERNAL ISSUE TRACKER",
+        description:
+            "A web application to create issue and tasks for your team members also priotize, tag and assign.",
```
Comment (ozkalai) [inline documentation/src/assets/examples.ts:99]:
```suggestion
            "A web application to create issues and tasks for your team members also prioritize, tag and assign.",
```

**Review by ozkalai — APPROVED** [review]

**Pull request merged**